apisix/plugins/kafka-logger.lua

```diff
@@ -141,6 +153,10 @@ function _M.log(conf, ctx)
         return
     end
 
+    local api_ctx = {}
+    api_ctx.conf_type = ctx.conf_type
+    api_ctx.conf_id = ctx.conf_id
+    api_ctx.conf_version = ctx.conf_version
```

Comment:
Maybe we can create the producer here and pass it to the `func`. So there is no need to create a copy of ctx.

Reply:
got